### PR TITLE
Update runtime to gnome 42

### PR DESCRIPTION
**Pull request by FakeShemp**

### CODE REVIEW

**Comment by FakeShemp** [comment]
Image links are broken at https://bluesabre.org

**Comment by TheEvilSkeleton** [comment]
@FakeShemp thanks! I opened https://github.com/bluesabre/menulibre/pull/97, In the meantime, could you patch the appstream file to make sure everything is working well?

**Comment by FakeShemp** [comment]
bot, build

File: org.bluesabre.MenuLibre.yaml [hunk 1]
```diff
@@ -103,5 +103,7 @@ modules:
       - type: git
         url: https://github.com/bluesabre/menulibre.git
         commit: cdde88daa6bf70632781870544357dc2f9a24006
+      - type: patch
+        path: 9097faf0965e83312f9fd6950efa35a754e2fd9d.patch
```
Comment (TheEvilSkeleton) [inline org.bluesabre.MenuLibre.yaml:107]: Can you rename the file to `0001-Fix-appstream.patch`?

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

**Review by TheEvilSkeleton — APPROVED** [review]
Thanks!

**Pull request merged**